Cmdline/Action/List.cs

```diff
@@ -31,66 +47,117 @@ public int RunCommand(CKAN.KSP ksp, object raw_options)
                 user.RaiseMessage("Installed Modules:\n");
             }
 
-            var installed = new SortedDictionary<string, Version>(registry.Installed());
-
-            foreach (KeyValuePair<string, Version> mod in installed)
+            if (exportFileType == null)
             {
-                Version current_version = mod.Value;
+                var installed = new SortedDictionary<string, Version>(registry.Installed());
 
-                string bullet = "*";
-
-                if (current_version is ProvidesVersion)
-                {
-                    // Skip virtuals for now.
-                    continue;
-                }
-                else if (current_version is DllVersion)
-                {
-                    // Autodetected dll
-                    bullet = "-";
-                }
-                else
+                foreach (KeyValuePair<string, Version> mod in installed)
                 {
-                    try
-                    {
-                        // Check if upgrades are available, and show appropriately.
-                        CkanModule latest = registry.LatestAvailable(mod.Key, ksp.Version());
+                    Version current_version = mod.Value;
 
-                        log.InfoFormat("Latest {0} is {1}", mod.Key, latest);
+                    string bullet = "*";
 
-                        if (latest == null)
-                        {
-                            // Not compatible!
-                            bullet = "X";
-                        }
-                        else if (latest.version.IsEqualTo(current_version))
+                    if (current_version is ProvidesVersion)
+                    {
+                        // Skip virtuals for now.
```

Comment:
That's just a whitespace change, add `?w=1` to a compare URL to force GitHub to [ignore whitespace in the diff](https://github.com/KSP-CKAN/CKAN/pull/1099/files?w=1).